trace/data.go

```diff
@@ -48,10 +50,10 @@ func (tr *trace) prepareTraceData(ctx context.Context, request json.RawMessage,
 		ApplicationRuntimeVersion: plugin.ApplicationRuntimeVersion,
 		ApplicationTags:           map[string]interface{}{},
 
-		RootSpanId:      tr.span.rootSpanId,
-		StartTimestamp:  tr.span.startTime,
-		FinishTimestamp: tr.span.finishTime,
-		Duration:        tr.span.duration,
+		RootSpanID:      tr.rootSpan.Context().(ttracer.SpanContext).SpanID,
```

Comment:
Can we convert the right-hand side to a function to make it more understandable like getRootSpanId(spanCtx spanContext) int64